ajax/libs/stickybits/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "stickybits",
+  "filename": "stickybits.js",
+  "description": "Stickybits is a lightweight alternative to `position: sticky` polyfills",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dollarshaveclub/stickybits.git"
+  },
+  "keywords": [
+    "stick",
+    "fixed",
+    "sticky",
+    "position",
+    "navigation",
+    "nav",
+    "dom",
+    "simple",
+    "javascript",
+    "stuck",
+    "waypoint",
+    "scroll",
+    "stickyheader",
+    "stickynavigation",
+    "fixedheader"
+  ],
+  "author": "Jeff Wainwright <jjwainwright2@gmail.com> (https://jeffry.in)",
+  "license": "MIT",
+  "homepage": "https://dollarshaveclub.github.io/stickybits/",
+  "npmName": "stickybits",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Would you mind telling us that can `stickybits.es.js` be used in web front-end?
If developers can use this file in front-end, then this pattern is good, else we have to exclude it by using `**/!(*.es.*)`.

Thank you.